bin/fusor-setup

```diff
@@ -43,6 +43,10 @@ Kafo::KafoConfigure.hooking.register_boot(:add_skip_hardware_check) do
     'SKIP_VALIDATIONS', 'Skip fusor-installer hardware validations.', :default => false
 end
 
+Kafo::KafoConfigure.hooking.register_boot(:add_upgrade_option) do
+  app_option '--upgrade',
+    :flag, 'Upgrade QCI to version 1.1', :default => false
```

Comment:
Should we consider removing the 1.1? Leave this text as "Upgrade QCI to next version"